pkg/types/constants.go

```diff
@@ -2,20 +2,18 @@ package types
 
 const (
 	// Environment Variable Constants
-	EnvAvpType            = "AVP_TYPE"
-	EnvAvpRoleID          = "AVP_ROLE_ID"
-	EnvAvpSecretID        = "AVP_SECRET_ID"
-	EnvAvpAuthType        = "AVP_AUTH_TYPE"
-	EnvAvpGithubToken     = "AVP_GITHUB_TOKEN"
-	EnvAvpK8sRole         = "AVP_K8S_ROLE"
-	EnvAvpK8sMountPath    = "AVP_K8S_MOUNT_PATH"
-	EnvAvpK8sTokenPath    = "AVP_K8S_TOKEN_PATH"
-	EnvAvpIBMAPIKey       = "AVP_IBM_API_KEY"
-	EnvAvpKvVersion       = "AVP_KV_VERSION"
-	EnvAvpPathPrefix      = "AVP_PATH_PREFIX"
-	EnvAWSAccessKey       = "AWS_ACCESS_KEY_ID"
-	EnvAWSSecretAccessKey = "AWS_SECRET_ACCESS_KEY"
-	EnvAWSRegion          = "AWS_REGION"
+	EnvAvpType         = "AVP_TYPE"
+	EnvAvpRoleID       = "AVP_ROLE_ID"
+	EnvAvpSecretID     = "AVP_SECRET_ID"
+	EnvAvpAuthType     = "AVP_AUTH_TYPE"
+	EnvAvpGithubToken  = "AVP_GITHUB_TOKEN"
+	EnvAvpK8sRole      = "AVP_K8S_ROLE"
+	EnvAvpK8sMountPath = "AVP_K8S_MOUNT_PATH"
+	EnvAvpK8sTokenPath = "AVP_K8S_TOKEN_PATH"
+	EnvAvpIBMAPIKey    = "AVP_IBM_API_KEY"
+	EnvAvpKvVersion    = "AVP_KV_VERSION"
+	EnvAvpPathPrefix   = "AVP_PATH_PREFIX"
+	EnvAWSRegion       = "AWS_REGION"
```

Comment:
If you decide to take the other suggestion then you can drop this variable too

Reply:
For most auth methods it's important to ensure a valid region is set though. I believe an env variable is the most common way of doing so. In light of this I'd  suggest to keep EnvAWSRegion as a dedicated constant, see also PR #155 .

Reply:
I'm convinced